.github/workflows/ci.yml

```
@@ -23,6 +23,7 @@
     outputs:
       CHANGED_APP_TARGETS: ${{ steps.generate_other_targets.outputs.CHANGED_APP_TARGETS }} 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)